docs/usage/getting_started.rst

```diff
@@ -82,23 +82,23 @@ biosamples config is a tsv separated file with the following columns
 #. Biosample 
 	- Name to associate with your sample. e.g K562
 #. DHS
-	- DNAse-seq BAM file (sorted w/ index file existence)
+	- DNAse-seq BAM file (sorted w/ .bai index file existence)
 #. ATAC
-	- ATAC-seq BAM/TagAlign file (sorted w/ index file existence)
+	- ATAC-seq BAM/TagAlign file (sorted w/ Tabix .tbi index file existence)
 #. H3K27ac
-	- H3K27ac ChIP seq BAM file
+	- H3K27ac ChIP seq BAM file (sorted w/ .bai index file existence)
 #. default_accessibility_feature
-	- Either DHS or ATAC
+	- Either DHS or ATAC (*explain what this means*)
 #. HiC_dir
-	- HiC directory for the biosample cell type. If not provided, uses avg hi-c
+	- HiC directory for the biosample cell type. If not provided and using hg38, uses a default cell-type average Hi-C file (*anthony check behavior here - since we have average hi-c only for hg38 and hg19, selecting a different genome build should result in not using any average hi-c data*)
 #. HiC_type
-	- e.g juicebox, avg, bed
+	- e.g juicebox, avg, bed   (*explain what this means*)
 #. HiC_resolution (int)
-	- resolution of the HiC data
+	- resolution of the HiC data  (*explain what this means*)
 #. HiC_gamma (float)
-	- represents the HiC data powerlaw fit slope
+	- represents the HiC data powerlaw fit slope   (*explain what this means*)
```

Comment:
Maybe we can have a separate section in the Detailed Methods page to describe powerlaw and we can just link to it from here